docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,498 @@
+
+## Explore
+
+### Logs: Choose which fields to display in a log line
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+When you're browsing logs in Explore, you can now click eye icon within a row to replace the log line's contents with the value of just one or more of the log fields or labels. This is helpful for scanning through your logs.
+
+{{< figure src="/media/docs/grafana/log-field-picker-10-1.gif" max-width="750px" caption="Log rows menu" >}}
+
+### Logs: Improved rendering performance of log lines
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+Browsing log lines is faster than ever, after a series of performance optimizations to log-related components.
+
+### Logs: See more log lines in logs context
+
+<!-- Gabor Farkas, Sven Grossmann -->
+
+_Generally available in all editions of Grafana_
+
+Log context allows you to view additional lines surrounding a specific log entry. With this enhancement, you can access as many log lines as needed within the log context. As you scroll through the logs, Grafana dynamically loads more log lines, ensuring a seamless and continuous viewing experience.
+
+### Elasticsearch logs sample
+
+<!-- Gareth Dawson -->
+
+_Generally available in all editions of Grafana_
+
+For Elasticsearch metric queries in Explore, you can now see the sample of log lines that contributed to the displayed results. To see these logs, click the collapsed Logs sample panel under your graph or table panel. If you want to interact with the log lines or modify the log query, click the **Open logs in split view** button and the log query will be executed in the split view.
+
+### Panel plugins in Explore
+
+<!-- Ben Donnelly -->
+
+_Experimental in all editions of Grafana_
+
+Data source plugin developers can now use any plugin to visualize data in Explore. Similar to `preferredVisualizationType`, we've introduced an experimental API to render visualizations by plugin ID. In the returned data frame, set the meta option `preferredVisualisationPluginId` to the plugin ID you want to be used when showing the data for given data frame.
+
+## Alerting
+
+_All Alerting features are generally available in all editions of Grafana_
+
+We’ve made a number of improvements to simplify the alert rule creation process as well as improvements to contact points and alert management. For all the details, refer to our [Alerting documentation]({{< relref "../alerting" >}}).
+
+### Alert rules
+
+We’ve made the following changes to alert rules.
+
+#### Alert instance routing preview
+
+_This feature is for Grafana-managed alert rules only._
+
+Preview how your alert instances will be routed if they fire while you're creating your alert rule. View routing for each Alertmanager you have configured to receive Grafana-managed alerts and if required, you can easily make adjustments to your custom labels to change the way your alert instances are routed.
```

Comment:
```suggestion
Preview how your alert instances will be routed if they fire while you're creating your alert rule. You can view routing for each Alertmanager you have configured to receive Grafana-managed alerts, and if required, easily make adjustments to your custom labels to change the way your alert instances are routed.
```

Reply:
sounds good to me-feel free to commit the change.